CombinatorialGames/Game/Small.lean

```diff
@@ -0,0 +1,109 @@
+/-
+Copyright (c) 2025 Tristan Figueroa-Reid. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Tristan Figueroa-Reid
+-/
+import CombinatorialGames.Surreal.Basic
+
+/-!
+# Small games all around
+
+We define dicotic games, games `G` where both players can move from
+every nonempty subposition of `G`. We prove that these games are small, and relate them
+to infinitesimals.
+
+## TODO
+
+- Define infinitesimal games as games x such that ∀ y : ℝ, 0 < y → -y < x ∧ x < y
+  - (Does this hold for small infinitesimal games?)
+- Prove that any short game is an infinitesimal (but not vice versa! see 1/ω)
+-/
+
+namespace IGame
+
+/-! # Dicotic Games -/
+
+private def DicoticAux (x : IGame) : Prop :=
+  (x.leftMoves = ∅ ∧ x.rightMoves = ∅) ∨
+  (x.leftMoves ≠ ∅ ∧ x.rightMoves ≠ ∅ ∧
+    (∀ l ∈ x.leftMoves, DicoticAux l) ∧
+    ∀ r ∈ x.rightMoves, DicoticAux r)
```

Comment:
I think it might be more natural to write this in terms of [`Xor'`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Logic/Basic.html#Xor'). Something like:
```suggestion
  Xor' (x.leftMoves = ∅) (x.rightMoves = ∅) ∧ 
    (∀ l ∈ x.leftMoves, DicoticAux l) ∧ (∀ r ∈ x.rightMoves, DicoticAux r)
```

Reply:
Am I reading this right? This looks like `x.leftMoves = ∅` xor `x.rightMoves = ∅` and ... which isn't the Dicotic definition.

Reply:
Oh you're right, xor is the negation of the predicate you want. So what about this?
```suggestion
  x.leftMoves = ∅ ↔ x.rightMoves = ∅ ∧ 
    (∀ l ∈ x.leftMoves, DicoticAux l) ∧ (∀ r ∈ x.rightMoves, DicoticAux r)
```